node/src/cli.rs

```diff
@@ -113,6 +113,7 @@ pub enum Subcommand {
 	Revert(sc_cli::RevertCmd),
 
 	/// The custom benchmark subcommmand benchmarking runtime pallets.
+	#[cfg(feature = "runtime-benchmarks")]
```

Comment:
Please also bump the version of all the crates that has been changed + runtime version.